specification/2.0/schema/mesh.primitive.schema.json

```diff
@@ -17,7 +17,7 @@
         "indices": {
             "allOf": [ { "$ref": "glTFid.schema.json" } ],
             "description": "The index of the accessor that contains the indices.",
-            "gltf_detailedDescription": "The index of the accessor that contains mesh indices.  When this is not defined, the primitives should be rendered without indices using `drawArrays()`.  When defined, the accessor must contain indices: the `bufferView` referenced by the accessor must have a `target` equal to 34963 (ELEMENT_ARRAY_BUFFER); a `byteStride` that is tightly packed, i.e., 0 or the byte size of `componentType` in bytes; `componentType` must be 5121 (UNSIGNED_BYTE), 5123 (UNSIGNED_SHORT) or 5125 (UNSIGNED_INT), the latter is only allowed when `OES_element_index_uint` extension is used; `type` must be `\"SCALAR\"`."
+            "gltf_detailedDescription": "The index of the accessor that contains mesh indices.  When this is not defined, the primitives should be rendered without indices using `drawArrays()`.  When defined, the accessor must contain indices: the `bufferView` referenced by the accessor should have a `target` equal to 34963 (ELEMENT_ARRAY_BUFFER); `componentType` must be 5121 (UNSIGNED_BYTE), 5123 (UNSIGNED_SHORT) or 5125 (UNSIGNED_INT), the latter may require enabling additional hardware support; `type` must be `\"SCALAR\"`."
```

Comment:
I tried to evaluate other usages, but I don't think they are useful:

- Interleaving index buffer data isn't supported by GPUs.
- Interleaving images makes no sense at all.
- Interleaving animation inputs/outputs or sparse indices with sparse values could work, but complicates implementation w/o any clear benefit (or maybe I'm wrong and it exists?)
- Interleaving skin's IBM matrices doesn't seem to be reasonable, too.

The main intent of #827 changeset was to simplify Vulkan vertex bindings by requiring the same stride value across all accessors that use the same buffer view.

Is it clear, that in case of omitted (zero) `bufferView.byteStride` that restriction still applies? Otherwise, I can elaborate a bit more on it in the README.